src/main/java/io/seqera/tower/cli/responses/labels/ManageLabels.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.responses.labels;
+
+import io.seqera.tower.cli.responses.Response;
+
+public class ManageLabels extends Response {
+
+    String operation;
+    String type;
```

Comment:
I think that these two properties won't be shown at the JSON output response if they are private